api/src/Controller/Stores/BaseProduct.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller\Stores;
+
+use Atlas\Query\Select;
+
+use Slim\Container;
+use Slim\Http\Request;
+use Slim\Http\Response;
+use App\Controller\BaseController;
+
+abstract class BaseProduct extends BaseController
+{
+    
+    /**
+     * @var int
+     */
+    protected $id = 0;
+
+
+    public function __construct(Container $container)
+    {
+        parent::__construct('products', $container);
```

Comment:
plural again.

Reply:
And staying that way :-) The others should all be corrected.